index.html

```diff
@@ -16,9 +16,10 @@
     <link rel="stylesheet" href="bower_components/font-awesome/css/font-awesome.min.css">
     <link rel="stylesheet" href="bower_components/seiyria-bootstrap-slider/dist/css/bootstrap-slider.min.css">
     <link rel="stylesheet" href="css/styles.css">
-  
     <script src="bower_components/jquery/dist/jquery.min.js"></script>
     <script src="bower_components/seiyria-bootstrap-slider/dist/bootstrap-slider.min.js"></script>
+    <script type="text/javascript">window._trackJs = { token: 'f58d0b7c4b4248939fddc582f9e55c9d' };</script>
```

Comment:
Can we move this line of JavaScript to `everything.js`?
I'm wondering if it needs to be the first line in the file, or inside `$(function() { ...` around line 361